src/firebase.js

```diff
@@ -0,0 +1,107 @@
+import firebase from 'firebase';
```

Comment:
this file was refactored out from main.js. It includes the setup and all helper functions for firebase.